assets/js/components/event/MobileBottomPanel.tsx

```diff
@@ -28,9 +32,12 @@ export const MobileBottomPanel = ({
   isChatLoaded,
   isBannedFromChat,
   eventTitle,
+  enablePictureInPicture,
 }: Props) => {
+  const { device } = useContext(ScreenTypeContext);
   const isOpen = card !== "hidden";
 
+  if (device !== "mobile") return null;
```

Comment:
I think `const { device } = useContext(ScreenTypeContext);` and `if (device !== "mobile") return null;` are here by mistake. We could remove them